client/signup/steps/site-title/style.scss

```diff
@@ -0,0 +1,81 @@
+.signup-site-title {
+	padding: 0;
+	position: relative;
+	border: none;
+	background: none;
+	box-shadow: none;
+	max-width: 600px;
+
+	// Not sure what this is about...
+	.validation-fieldset__validation-message {
+			display: none;
+	}
+
+	.signup-site-title__label {
+		display: none;
+	}
+
+	.signup-site-title__input {
+		padding: 14px 16px;
+	}
+
+	.signup-site-title__button {
+		position: absolute;
+			top: 7px;
+			right: 8px;
+	}
+}
+
+.example-site-title {
+	box-sizing: border-box;
+	position: relative;
+	padding-top: 16px;
+	padding-right: 348px;
+	max-width: 600px;
+	margin: 30px auto 60px auto;
+
+	p {
```

Comment:
Would be good to have classes for the generic html selectors if we control the markup.